backend/src/main/java/com/bakdata/conquery/apiv1/QueryProcessor.java

```diff
@@ -170,12 +168,20 @@ private void translateToOtherDatasets(Dataset dataset, QueryDescription query, U
 
 	public FullExecutionStatus getStatus(ManagedExecution<?> query, UriBuilder urlb, User user) {
 		query.initExecutable(datasetRegistry, config);
-		return query.buildStatusFull(storage, urlb, user, datasetRegistry, AuthorizationHelper.buildDatasetAbilityMap(user,datasetRegistry));
+		return query.buildStatusFull(storage, urlb, user, datasetRegistry, AuthorizationHelper.buildDatasetAbilityMap(user, datasetRegistry));
 	}
 
 	public FullExecutionStatus cancel(User user, Dataset dataset, ManagedExecution<?> query, UriBuilder urlb) {
 		// TODO implement query cancel functionality
 		user.authorize(query, Ability.CANCEL);
 		return null;
 	}
+
+	public FullExecutionStatus reexecute(User user, ManagedExecution<?> query, UriBuilder responseBuilder) {
+		if(query.getState().equals(ExecutionState.RUNNING))
+
+		ExecutionManager.execute(getDatasetRegistry(), query, config);
+
+		return query.buildStatusFull(storage, responseBuilder, user, getDatasetRegistry(), AuthorizationHelper.buildDatasetAbilityMap(user, getDatasetRegistry()));
+	}
```

Comment:
Dem `if` fehlen die Klammern: Die Query wird nur Reexecuted, wenn sie gerade RUNNING ist? zu mindest in C wäre es so

Reply:
Bitte noch einen Test für den Endpunkt

Reply:
done!